File: Selenium/Project/Activity6.java
package CRMProject;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterTest;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.Test;

public class Activity6 {
    WebDriver driver;

    @BeforeTest
    public void beforeMethod() {
        // Set up the Firefox driver
        WebDriverManager.firefoxdriver().setup();
        //Create a new instance of the Firefox driver
        driver = new FirefoxDriver();
        //Open the browser
        driver.get("https://alchemy.hguy.co/crm/");
    }

    @Test
    public void MenuCheck() {
        driver.findElement(By.id("user_name")).sendKeys("admin");
        driver.findElement(By.id("username_password")).sendKeys("pa$$w0rd");
        driver.findElement(By.xpath("//*[@id=\"bigbutton\"]")).click();
        String navigationMenu = driver.findElement(By.id("toolbar")).getAttribute("id");
        Assert.assertNotNull(navigationMenu);
        System.out.println("The user is on the home page with the Navigation Menu");
        String ActivitiesItem = driver.findElement(By.id("grouptab_3")).getAttribute("id");
        Assert.assertEquals("grouptab_3", ActivitiesItem);
        System.out.println("The item 'Activities' exists in the Menu");
    }

    @AfterTest
    public void afterMethod() {
        //Close the browser
        driver.quit();
    }

}

File: Selenium/Project/activity6.py
# Menu checking
#Goal: Make sure that the “Activities” menu item exists and is clickable

from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.remote.webdriver import By

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Initialize wait object
    wait = WebDriverWait(driver, 10)

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/crm/")
    print("Home Page Title is: " + driver.title)
    # Menu checking
    #Goal: Make sure that the “Activities” menu item exists and is clickable
    driver.find_element(By.ID, "user_name").send_keys("admin")
    driver.find_element(By.ID,"username_password").send_keys("pa$$w0rd")
    driver.find_element(By.XPATH,"//*[@id=\"bigbutton\"]").click()
    navigationMenu = driver.find_element(By.ID,"toolbar").get_attribute("id")
    print("The user is on the home page with the Navigation Menu")
    ActivitiesItem = driver.find_element(By.ID,"grouptab_3").get_attribute("id")
    assert ActivitiesItem == "grouptab_3"
    print("The item 'Activities' exists in the Menu")